compiler/aarch64/codegen/ARM64SystemLinkage.cpp

```diff
@@ -501,18 +501,18 @@ TR::ARM64SystemLinkage::createPrologue(TR::Instruction *cursor)
 void
 TR::ARM64SystemLinkage::createPrologue(TR::Instruction *cursor, List<TR::ParameterSymbol> &parmList)
    {
-   TR::CodeGenerator *codeGen = cg();
-   TR::Machine *machine = codeGen->machine();
+   TR::CodeGenerator *cg = cg();
```

Comment:
This will likely be a problem with the name shadowing.  You might need to say `this->cg()`.